apps/src/applab/applab.js

```diff
@@ -847,15 +847,38 @@ Applab.init = function(config) {
   }
 };
 
-Applab.appendToEditor = function(newCode) {
+/**
+ * @param {string} newCode Code to append to the end of the editor
+ */
+Applab.appendToEditor = function (newCode) {
   var code = studioApp.editor.addEmptyLine(studioApp.editor.getValue()) + newCode;
   studioApp.editor.setValue(code);
 };
 
+Applab.scrollToEnd = function () {
```

Comment:
Yeah, I think because we're reaching into the internals of "which editor is currently active?" this does feel like it might be better as a core droplet feature.

Maybe the API is a `scrollToLine` that works regardless of the current mode?